### PR TITLE
Feature/add xml attributes

### PR DESCRIPTION
**Pull request by vinzlange**

### CODE REVIEW

File: _docs_integrate/63-attribute-values.md [hunk 1]
```diff
@@ -449,6 +449,18 @@ The officially registered pseudonym of a person.
...
+## SchematizedXML
+
+SchematizedXML can be used to exchange files in XML format. The exchange of XML files is also possible via [`FileReference`](#filereference), but SchematizedXML has the advantage that it is possible to validate the XML and display the attributes in the wallet.
+
+**Properties**
+
+| Name        | Type               | Required | Validation                                                 |
+| ----------- | ------------------ | :------: | ---------------------------------------------------------- |
+| `@type`     | `"SchematizedXML"` |    ✓     |                                                            |
+| `value`     | `string`           |    ✓     | max. length: 50000 <br>must be a valid XML                 |
```
Comment (jkoenig134) [inline _docs_integrate/63-attribute-values.md:461]: isn't it an `XML document` instead of only XML?

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

File: _docs_integrate/63-attribute-values.md (hunk 1 above)
Comment (vinzlange) [reply _docs_integrate/63-attribute-values.md:461]: plain xml string, not a file.

Comment (jkoenig134) [reply _docs_integrate/63-attribute-values.md:461]: xml is a format, I dont think there is `an xml`, right?

**Review by jkoenig134 — APPROVED** [review]

File: _docs_integrate/63-attribute-values.md (hunk 1 above)
Comment (vinzlange) [reply _docs_integrate/63-attribute-values.md]: wouldn't the same be true for url format?

Comment (jkoenig134) [reply _docs_integrate/63-attribute-values.md]: IMO we talk about URLs in another way than "XMLs". But maybe it's just my gusto.

**Pull request merged**